book/glossary.md

```diff
@@ -60,21 +73,26 @@ __WHATWG__: Web Hypertext Application Technology Working Group. The
 standardization organization for __HTML__, __DOM__, and a few other key web
 APIs.
 
-
-
-
-Document model
-==============
+Web Documents
+=============
 
 __Animation__: A sequence of visual changes on a computer screen
 interpreted by humans to look like movement.
 
-__CSS__: Cascading Style Sheet. A format for storing rules that specify the
+__Attribute__; A parameter on an __element__ indicating some information,
```

Comment:
```suggestion
__Attribute__: A parameter on an __element__ indicating some information,
```

Also perhaps you mean HTML attribute? Because CSS attributes also exist but I'm not sure this definition exactly applies. (Pseudo-elements can have attributes.)